service/src/main/java/com/epam/esm/service/service/order/OrderServiceImpl.java

```diff
@@ -135,33 +110,35 @@ public OrderFinder getFinder() {
     }
 
     @Override
-    public List<Order> read(MultiValueMap<String, String> params) throws NotFoundException, BadRequestException {
+    public List<Order> findByParameters(MultiValueMap<String, String> params, Pageable pageable)
+            throws NotFoundException, BadRequestException {
         OrderFinder finder = getFinder();
         for (Map.Entry<String, List<String>> entry : params.entrySet()) {
             try {
                 validateParameterValues(entry.getValue());
-                parseParameter(finder, entry.getKey(), entry.getValue());
+                if (!PageableParameters.contains(entry.getKey())) {
+                    parseFindParameter(finder, entry.getKey(), entry.getValue());
+                }
             } catch (IllegalArgumentException e) {
                 throw new BadRequestException(e, ErrorCodes.ORDER_BAD_REQUEST);
             }
         }
-        return readBy(finder);
+        return readBy(finder, pageable);
```

Comment:
please rename it to findByFinder so it would be consistent with other services

Reply:
done